meetings/2021-03-22.md

```diff
@@ -0,0 +1,91 @@
+# 2021-03-22 Authentication Panel
+
+https://meet.jit.si/solid-authentication
+
+## Agenda
+
+* [Meeting notes for 2021-03-15 #147](https://github.com/solid/authentication-panel/pull/147) (1m)
+* Any discussion/follow-ups to WICG/SolidCG meeting
+   * re WICG https://github.com/WICG/WebID/issues/54
+    * WICG (Updated README as requested: https://github.com/WICG/WebID/commit/9ae80cc3e78c994bd555363343103ecaf476da98#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 )
+    * Open question to document: What are Solid's needs that can be passed to WICG
+* [Proposal: change webid claim to solid #146](https://github.com/solid/authentication-panel/issues/146)
+* [specify parameters in WWW-Authenticate header #152](https://github.com/solid/authentication-panel/issues/152)
+* [Solid OIDC Conformance Classes #133](https://github.com/solid/authentication-panel/issues/133)
+* [Clarifying specific areas of overlap with AuthZ #145](https://github.com/solid/authentication-panel/issues/145)
+* [keyId's do not exactly refer to keys anymore #151](https://github.com/solid/authentication-panel/issues/151)
+
+## Present
+
+* Henry Story
+* Aaron Coburn
+* Matthieu Bosquet
+* Sarven Capadisli
+* Elf Pavlik
+* Benoit Alessandroni
+
+## Minutes
+
+### Any discussion/follow-ups to WICG/SolidCG meeting
+
+Sarven: They added note to README mentioning name collision.
+...: We should come back to them with what solid would need from web browsers. Create issue?
+
+Henry: Good idea to create an issue. It need to first implement HTTPSig, ACLs and then get to Credentials before I can start constructively looking at how we could work together. So my guess this will take 3 months before  I can give an informed answer.
```

Comment:
```suggestion
Henry: Good idea to create an issue. I need to first implement HTTPSig, ACLs, and then get to Credentials, before I can start constructively looking at how we could work together. So my guess is this will take 3 months before I can give an informed answer.
```